File: Lab4/app/src/main/java/com/example/pamo/lab4/quiz/Quiz.java
package com.example.pamo.lab4.quiz;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class Quiz implements Serializable {

    private static final long serialVersionUID = 3379200832549055538L;

    private String name;
    private List<Question> questions = new ArrayList<>();
    private int score;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<Question> getQuestions() {
        return questions;
    }

    public void setQuestions(List<Question> questions) {
        this.questions = questions;
    }

    public int getScore() {
        return score;
    }

    public void setScore(int score) {
        this.score = score;
    }
}

File: Lab4/app/src/main/java/com/example/pamo/lab4/quiz/Quiz.kt
package com.example.pamo.lab4.quiz

import java.io.Serializable
import java.util.*

class Quiz : Serializable {
    var name: String? = null
    var questions: MutableList<Question> = ArrayList()

    companion object {
        private const val serialVersionUID = 3379200832549055538L
    }
}